daemon/indexing/lib/search.js

```diff
@@ -113,16 +115,74 @@ class Listing {
         id: hit._id,
         name: hit._source.name,
         description: hit._source.description,
-        price: hit._source.price,
+        priceEth: hit._source.priceEth,
       }
       listings.push(listing)
     })
     return listings
   }
 }
 
+class Offer {
+  /*
```

Comment:
nit: JSdoc comments actually start with "/**" - my bad for using "/*" when I created this file...
Also it is @param vs @params... again my bad :(

Reply:
Will update.